src/4cade.init.a

```diff
@@ -166,6 +166,15 @@
          dex
          bpl   -
 
+         jsr   DisableAccelerator    ; cycle counting requires 1MHz
+         lda   #0
+         sta   MockingboardSlot      ; default to no Mockingboard
+         jsr   HasMockingboard
+         bcc   +
+         stx   MockingboardSlot      ; save mockingboard slot in LC RAM
```

Comment:
rename `HasMockingboard` to `GetMockingboardSlot` and change it to just return a slot number or `#$00` (and don't set/clear to carry), then unconditionally `stx MockingboardSlot` here.

Reply:
Done